torch/csrc/jit/interned_strings.h

```diff
@@ -1 +1,11 @@
 #include <ATen/core/interned_strings.h>
+
+namespace torch {
+namespace jit {
+namespace prim = ::c10::prim;
```

Comment:
OOC, is there a reason to prefer this over 'namespace prim { using namespace c10::prim }'?

Reply:
I'll do that